drivers/store/redis/store.go

```diff
@@ -52,258 +91,104 @@ func NewStoreWithOptions(client Client, options limiter.StoreOptions) (limiter.S
 	if store.MaxRetry <= 0 {
 		store.MaxRetry = 1
 	}
-
 	return store, nil
 }
 
+// preloadLuaScripts would preload the  incr and peek lua script
+func (store *Store) preloadLuaScripts(ctx context.Context) error {
+	incrLuaSHA, err := store.client.ScriptLoad(ctx, luaIncrScript).Result()
+	if err != nil {
+		return fmt.Errorf("load incr lua script err: %w", err)
```

Comment:
Could you wrap the error using `errors.Wrap`, we would like to keep the stacktrace.

Reply:
Fixed